### DISCUSSION TITLE
How to customize unit?

### DISCUSSION THREAD

**Discussion by xmsz** [unocss/unocss#562] [Q&A] [answered]
```
'w-10rpx' => .w-10rpx { wdith: 10rpx }

```

**Comment by Arhell** [comment]
https://github.com/unocss/unocss#features  =>**Fully customizable**

https://github.com/unocss/unocss#configurations

https://github.com/unocss/unocss#custom-rules

https://github.com/unocss/unocss#custom-variants

**Comment by chu121su12** [comment, marked as answer]
Closed in #563
